strings/traumas.json

```diff
@@ -1,153 +1,111 @@
 {
     "brain_damage": [
-        "@pick(semicolon)IM A PONY NEEEEEEIIIIIIIIIGH",
-        "without oxigen blob don't evoluate?",
-        "@pick(semicolon)CAPTAINS A COMDOM",
-        "@pick(semicolon) @pick(george) @pick(mellens) is grifing me HALP!!!",
-        "can u give me @pick(mutations)?",
-        "THe saiyans screwed",
-        "Bi is THE BEST OF BOTH WORLDS>",
-        "@pick(semicolon)I WANNA PET TEH monkeyS",
-        "stop grifing me!!!!",
-        "SOTP IT#",
-        "shiggey diggey!!",
-        "@pick(semicolon)A PIRATE APPEAR",
-        "FUS RO DAH",
-        "fucking 4rries!",
-        "stat me",
-        ">my face",
-        "roll it easy!",
-        "waaaaaagh!!!",
-        "red wonz go fasta",
-        "FOR TEH EMPRAH",
-        "lol2cat",
-        "dem dwarfs man, dem dwarfs",
-        "SPESS MAHREENS",
-        "hwee did eet fhor khayosss",
-        "lifelike texture ;_;",
-        "luv can bloooom",
```

Comment:
dont see why a LOT of these got changed. What's wrong with "luv can bloooom"? That sounds like the perfect example of brain-damage

Other notable examples - "SOTP IT", "hwat dose tha @pick(random_gibberish) mean?????", "youed call her a toeugh bithc", "AI laW 22 Open door", this SI mY stATIon......", Among many, many others

While I'm all for replacing the edgy ones or meta ones ("lol2cat", "REVIRT COBY CHEM!!!!!!!!", "@pick(semicolon)the ai and borgs are mettacomming I think", similar memes), re-writing all of them to feel more like someone who's just socially anxious instead of someone with literal brain damage feels... lame, to be honest. Like, people are crying 'de-souling' but it literally DOES feel soul-less and un-fun.